src/App.jsx

```diff
@@ -22,7 +24,11 @@ function App() {
 
   return (
     <>
+    <BrowserRouter>
```

Comment:
Can you run a formatter on this code so that we have proper indentation? I would recommend installing the `Prettier` extension and turning on `formatOnSave` and `autoSave` in your VS Code settings.

As an aside, I think we should set up a `.prettierrc` file so everyone's formatter uses the same rules, but we can do that in a separate PR.